src/views/MainView/MainView.jsx

```diff
@@ -1,6 +1,14 @@
 
 function MainView(){
-    return <section data-testid="main-view"></section>
+    const mockData = require('../../mocks/youtube-videos-mock.json');
+
+    return <section data-testid="main-view">
+        <div>
+            {mockData.items.map((item, idx) => {
+                return <div key={idx} data-testid="video-container">{item.snippet.title}</div>
```

Comment:
Se recomienda que el el `key` cuando se tiene una lista de varios elementos no sea el número de su index, sino un string que identifique de manera estable ese item.
https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Fue corregido en el proceso de refactorización en el commit 3a039bf :blush: